### PR TITLE
Added collectd role

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: roles/collectd/tasks/defaults.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      port: '123'
+      ReverseLookups: 'false'
+    syslog:
+      loglevel: 'info'
+    write_graphite:
+      carbon: 'monitoring'
+      port: 2003
+      protocol: 'udp'
+      escapecharacter: '.'
+      seperateinstances: 'true'
```
Comment (Azulinho) [inline roles/collectd/tasks/defaults.yml]: https://www.grammarly.com/blog/separate-seperate/

File: roles/collectd/tasks/defaults.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      host: 'localhost'
+      port: '123'
+      ReverseLookups: 'false'
+    syslog:
+      loglevel: 'info'
+    write_graphite:
+      carbon: 'monitoring'
+      port: 2003
+      protocol: 'udp'
+      escapecharacter: '.'
```
Comment (Azulinho) [inline roles/collectd/tasks/defaults.yml]: escape_character please instead of escapecharacter (as per the rest of the keys in this dict)

Comment (Azulinho) [reply roles/collectd/tasks/defaults.yml]: actually can we make sure we're consistent ?
some are camelcase, others are snake, others nothing at all

File: roles/collectd/tasks/defaults.yml (hunk 1 above)
Comment (brettminnie) [reply roles/collectd/tasks/defaults.yml]: Take this up with collectd, it's their variable name :)

Kidding, was lifted from our old role

File: roles/collectd/tasks/defaults.yml (hunk 2 above)
Comment (Azulinho) [reply roles/collectd/tasks/defaults.yml]: oh I see, they're actually collectd/graphite parameters. ok ignore me then

**Review by Azulinho — APPROVED** [review]

**Pull request merged**